tango_simlib/sim_xmi_parser.py

```diff
@@ -256,6 +318,8 @@ def extract_command_description_data(self, description_data):
         output_parameter = description_data.find('argout')
         command_data['argoutDescription'] = output_parameter.attrib['description']
         command_data['argoutType'] = self._get_arg_type(output_parameter)
+        command_data['inherited'] = (
+                description_data.find('status').attrib['inherited'])
         return command_data
```

Comment:
@lvdheever That won't be necessary. When we parse the file, we get the complete `tree` object (`tree = xml.etree.ElementTree.parse()`) with all the fields intact, which is all we need to overwrite the XMI file. After we identify the parts of the tree that need to be pruned we just need to call tree.write() and all that information with be written to the xmi file. 

All we need is to get hold of that `tree` object.